src/utils/constants.ts

```diff
@@ -11,15 +11,15 @@ export enum Network {
 export const NETWORK = process.env.NETWORK.toLowerCase();
 export const API_KEY = process.env.INFURA_API_KEY;
 
-export const NETWORK_URL = `https://${NETWORK}.infura.io/v3/${API_KEY}`;
-
-const SAFE_ENDPOINT_URLS = {
+export const SAFE_ENDPOINT_URLS = {
   [Network.rinkeby]: `https://safe-transaction.rinkeby.gnosis.io`,
   [Network.mainnet]: `https://safe-transaction.gnosis.io`,
 };
 
-export const SAFE_ENDPOINT_URL =
-  SAFE_ENDPOINT_URLS[Network[NETWORK]] || SAFE_ENDPOINT_URLS[Network.rinkeby];
+export const INFURA_ENDPOINT_URL = {
+  [Network.rinkeby]: `https://rinkeby.infura.io/v3/${API_KEY}`,
+  [Network.mainnet]: `https://mainnet.infura.io/v3/${API_KEY}`,
+};
```

Comment:
What about exporting this as a function, since infura endpoint is the same for all networks, even xDai:


```suggestion
export function getInfuraEndpoint(network:string):string {
  if (!Network[network]) {
    throw new Error(`Network ${network} not registered for Infura`)
  }
  return `https://${network}.infura.io/v3/${API_KEY}`
};
```

Disclaimer: code not tested nor linted